broker/bucketbroker/cmd/bucketbroker/app/app.go

```diff
@@ -33,6 +39,9 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Kubeconfig, "kubeconfig", o.Kubeconfig, "Path pointing to a kubeconfig file to use.")
 	fs.StringVar(&o.Address, "address", "/var/run/iri-bucketbroker.sock", "Address to listen on.")
 
+	fs.Float32VarP(&o.QPS, "qps", "", defaults.QPS, "Kubernetes client qps.")
```

Comment:
As we are not using shortnames for all the other flags (for now) I would use here `Float32Var` instead of `Float32VarP`.